### ISSUE TITLE
SIGSEGV when using vulkan without specifying gpu-context

### ISSUE THREAD

**Issue by LaserEyess** [mpv-player/mpv#4981]
### mpv version and platform

```
$ mpv --version
mpv 0.27.0-228-gb4c1f0aae3 (C) 2000-2017 mpv/MPlayer/mplayer2 projects
 built on Thu Oct 12 17:05:44 EDT 2017
ffmpeg library versions:
   libavutil       55.58.100
   libavcodec      57.89.100
   libavformat     57.71.100
   libswscale      4.6.100
   libavfilter     6.82.100
   libswresample   2.7.100
ffmpeg version: 3.3.4
```

That version name isn't completely accurate, and an artifact of the AUR PKGBUILD I used, inside the source directory: 
```
$ git rev-parse HEAD
b4c1f0aae372df61ace6e179faba75746dff74a5
```
### Reproduction steps

mpv --no-config --vo=gpu --gpu-api=vulkan --spirv-compiler=shaderc $file 

(this works with any file I have tested)

### Expected behavior
mpv to play normally

### Actual behavior
mpv segfaults without --gpu-context=x11

### Log file
Here's a log file with -v 
https://0x0.st/Cum.txt

Here's my poor attempt at a gdb backtrace, it includes a run with --gpu-context and one without
https://0x0.st/Cua.txt

### Sample files

any video file reproduces it

**Comment by atomnuker** [comment]
Should be fixed in 436082e5076

**Comment by LaserEyess** [comment]
This indeed fixes my issue, thanks
